docker-run-sync-rapid-pro-to-engagement-db.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
I think this can be called just docker-sync-rapid-pro-to-engagement-db i.e. I think -run- is unnecessary

Reply:
Agreed & updated.